bootstrap.py

```diff
@@ -0,0 +1,39 @@
+import base64
```

Comment:
A short comment at the top of this file would be useful, just to outline what it does. Maybe a python3 shebang too.